hoot-core-test/src/test/cpp/hoot/core/io/OsmXmlReaderTest.cpp

```diff
@@ -64,9 +64,6 @@ class OsmXmlReaderTest : public CppUnit::TestFixture
 
         CPPUNIT_ASSERT_EQUAL(36, (int)map->getNodes().size());
         CPPUNIT_ASSERT_EQUAL(4, (int)map->getWays().size());
-
-        OsmXmlWriter writer;
-        writer.write(map, "output.osm");
```

Comment:
There are quite a few tests that write to `output.osm` but don't do anything with it.  This isn't necessary and only slows things down.  There are unit tests to make sure that `OsmXmlWriter` does what it ought to do.